hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -158,6 +189,294 @@ const KEY_TABLE_COLUMNS = [
   }
 ];
 
+const UCOLUMNS = [
```

Comment:
I checked and the `UCOLUMNS`, `OCOLUMNS`, `MCOLUMNS` and `ACOLUMNS` are exactly the same. Why do we have it copied 4 times? 
If we want to handle the `MISSING` state the same as the others we could have only the COLUMNS (starts on line 97) and adjust it to the parameters.

Reply:
I have used another REPLICATEDCOLUMNS as one columns differs Unhealthy Since.

![image](https://user-images.githubusercontent.com/112169209/196420218-8eb1696d-da8f-484a-a781-ace8784ff624.png)

Reply:
Done  Code Changes